src/styles/app.scss

```diff
@@ -616,3 +616,13 @@ $border-colour: govuk-colour("grey-2");
   margin: 0; 
 }
 
+.govuk-fieldset{
+  &.horizontalChoices{
```

Comment:
should ideally be kebab-case

Reply:
I've raised new PR's to update it:
https://github.com/UKHomeOffice/hocs-frontend/pull/379
https://github.com/UKHomeOffice/hocs-data-wcs/pull/59